src/routes/projects.rs

```diff
@@ -551,6 +552,41 @@ pub async fn project_edit(
                     }
                 }
 
+
+                let user_id: database::models::ids::UserId = user.id.into();
+                let notified_members = sqlx::query!(
+                    "
+                    SELECT tm.user_id id
+                    FROM team_members tm
+                    WHERE tm.team_id = $1 AND tm.user_id <> $2
+                    ",
+                    project_item.inner.team_id as database::models::ids::TeamId,
+                    user_id as database::models::ids::UserId
+                )
+                .fetch_many(&mut *transaction)
+                .try_filter_map(|e| async { Ok(e.right().map(|c| database::models::UserId(c.id))) })
+                .try_collect::<Vec<_>>()
+                .await?;
+
+                NotificationBuilder {
+                    notification_type: Some("status_update".to_string()),
+                    title: format!("**{}**'s status has changed!", project_item.inner.title),
+                    text: format!(
+                        "The project {}'s status has changed from {} to {}",
+                        project_item.inner.title,
+                        project_item.inner.status.as_friendly_str(),
+                        status.as_friendly_str()
+                    ),
+                    link: format!(
+                        "/{}/{}",
+                        project_item.project_type,
+                        ProjectId::from(id)
+                    ),
+                    actions: vec![],
+                }
+                .insert_many(notified_members, &mut transaction)
+                .await?;
```

Comment:
It only really needs to be sent if the status was set by a mod, imo. Currently, it gets sent for any status change, including for example draft -> processing

Reply:
I've made it so that it only sends notifications if the user changing the status isn't in the team